search-commons/src/test/java/no/unit/nva/search/IndexingClientTest.java

```diff
@@ -0,0 +1,153 @@
+package no.unit.nva.search;
+
+import static no.unit.nva.search.ElasticSearchHighLevelRestClient.BULK_SIZE;
+import static no.unit.nva.search.IndexingConfig.objectMapper;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static nva.commons.core.attempt.Try.attempt;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNot.not;
+import static org.hamcrest.core.IsNull.nullValue;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.model.Organization;
+import no.unit.nva.model.Publication;
+import no.unit.nva.search.exception.SearchException;
+import no.unit.nva.search.models.EventConsumptionAttributes;
+import no.unit.nva.search.models.IndexDocument;
+import nva.commons.core.attempt.Try;
+import org.elasticsearch.action.DocWriteResponse;
+import org.elasticsearch.action.bulk.BulkRequest;
+import org.elasticsearch.action.bulk.BulkResponse;
+import org.elasticsearch.action.delete.DeleteResponse;
+import org.elasticsearch.action.update.UpdateResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+public class IndexingClientTest {
+
+    public static final int NUMBER_NOT_DIVIDABLE_BY_BLOCK_SIZE = 1256;
+
+    @Test
+    void shouldCreateDefaultObjectWithoutFailing() {
+        assertDoesNotThrow((Executable) IndexingClient::new);
+    }
+
+    @Test
+    void batchInsertIndexesAllDocumentsInBatchInBulksOfSpecifiedSize() throws IOException {
```

Comment:
All tests were moved to from the ElasticSearchHighLevelRestClient and the only difference is the name of the client, which now is "IndexingClient"